docs/NetCore_GettingStarted.md

```diff
@@ -0,0 +1,71 @@
+7. Make sure you have done everything from the [Definition of done list](#definition-of-done) below. 
+
+## Definition of done 
+
+- Analyzer implemented to work for C# and VB. 
+	- Unit tests for C# 
+		- All scenarios covered. 
+		- Prefer markup syntax for the majority of tests. 
+		- If your analyzer has placeholders in the diagnostic message and you want to test the arguments, write a smaller number of tests using the `VerifyCS.Diagnostic` syntax to construct specific diagnostic forms.
+	- Unit tests for VB. 
+		- Obvious positive and negative scenarios covered. 
+		- If the implementation uses any syntax-specific code, then all scenarios must be covered. 
+- Fixer implemented for C#, using the language-agnostic APIs if possible. 
+	- If the fixer can be entirely implemented with language-agnostic APIs `(IOperation)`, then VB support is essentially free. 
+	- With a language-agnostic fixer, apply the attribute to indicate the fixer also applies to VB and add mainline VB tests. 
+	- If language-specific APIs are needed to implement the fixer, the VB fixer is not required.
+	- Do not separate analyzer tests from code fix tests. If the analyzer has a code fix, then write all your tests as code fix tests.
+		- Calling VerifyCodeFixAsync(source, source) verifies that the analyzer either does not produce diagnostics, or produces diagnostics where no code fix is offered
+		- Calling VerifyCodeFixAsync(source, fixedSource) verifies the diagnostics (analyzer testing) and verifies that the code fix on source produces the expected output	
+- Run the analyzer locally against `dotnet/runtime` [instructions](#Testing-against-the-Runtime-repo). 
+	- Use the failures to discover nuance and guide the implementation details. 
+	- Run the analyzer against `dotnet/roslyn` [instruction](#Testing-against-the-Roslyn-repo), and with `dotnet/aspnetcore` if feasable. 
+	- Review each of the failures in those repositories and determine the course of action for each. 
+- Document for review: severity, default, categorization, numbering, titles, messages, and descriptions.. 
+- Document for review: matching and non-matching scenarios, including any discovered nuance. 
+- Document for the new analyzer and rules on [MicrosoftDocs](https://github.com/microsoftDocs/visualstudio-docs-pr/). (need permission, [instructions](#Adding-documentation-for-new-CA-rules)). Documentation PR must be submitted within **ONE WEEK** of the rule implementation being merged. 
+- PR merged into roslyn-analyzers. 
+- Failures in dotnet/runtime addressed. 
+
+## Testing against the Runtime repo 
+
+1. Navigate to the root of the Roslyn-analyzers repo and run these commands: 
+	- `cd roslyn-analyzers` 
+	- `set RUNTIMEPACKAGEVERSION=3.0.0` 
+	- `build.cmd -ci /p:AssemblyVersion=%RUNTIMEPACKAGEVERSION% /p:OfficialBuild=true`
```

Comment:
I am following these instructions and I noticed this step shows a lot of errors with this message:

```
D:\roslyn-analyzers\.packages\microsoft.dotnet.arcade.sdk\1.0.0-beta.20221.2\tools\Version.targets(15,5): error : AssemblyVersion '3.0.0'
overridden by auto-generated version [D:\roslyn-analyzers\nuget\Microsoft.CodeAnalysis.FxCopAnalyzers\Microsoft.CodeAnalysis.FxCopAnalyzer
s.Package.csproj]
```

The build finished and the DLLs were generated, though. The message is telling us the assembly version was overriden with 3.0.0, which we did in the previous step, but it's being logged as an error.

Can the error be ignored? If yes, can we make sure to add a comment after this step, explaining that the error will show up? cc @bartonjs

Reply:
strange i didn't see any error, or may not noticed if it was not showed in red

Reply:
The error does not seem to make the build fail, the DLLs ended up getting generated. I think we can add a message after this step:
```suggestion
	- `build.cmd -ci /p:AssemblyVersion=%RUNTIMEPACKAGEVERSION% /p:OfficialBuild=true`
	    - Note: If you get build errors about the assembly version being overridden, you can ignore them, since the DLLs will end up getting generated successfully. Error message example:
	        `\roslyn-analyzers\.packages\microsoft.dotnet.arcade.sdk\1.0.0-beta.20221.2\tools\Version.targets(15,5): error : AssemblyVersion '3.0.0'
overridden by auto-generated version [\roslyn-analyzers\nuget\Microsoft.CodeAnalysis.FxCopAnalyzers\Microsoft.CodeAnalysis.FxCopAnalyzer
s.Package.csproj]`
	    
```

Reply:
Mmm I built again and the error is gone. Not sure what happened. @buyaa-n you can ignore my suggestion.